Source/WebCore/platform/LocalizedStrings.cpp

```diff
@@ -503,6 +503,13 @@ String contextMenuItemTagTranslate(const String& selectedString)
 }
 #endif
 
+#if ENABLE(UNIFIED_PDF)
+String contextMenuItemPDFOpenWithPreview()
```

Comment:
I wish these functions had "Title" in the name; hard to know at the call site that this returns a string.

Reply:
I'll likely do a renaming patch once all the items are added since I'm planning on pulling in the other functions that are defined below